### PR TITLE
gpu support

### PR DESCRIPTION
**Pull request by 42Max42**
Modify python-env to allow GPU usage for maskrcnn and dextr.
If no gpu, also works with CPU.
To run with GPU see: https://github.com/NVIDIA/nvidia-docker

### CODE REVIEW

**Comment by jsbroks** [comment]
@42Max42 please revert to npm instead of yarn

File: backend/requirements.txt [hunk 1]
```diff
@@ -16,3 +16,25 @@ pytest-ordering==0.6
...
+Pillow
+cython
+matplotlib
+scikit-image
+keras==2.1.1
+h5py
+imgaug
+IPython[all]
+
+numpy
```
Comment (jsbroks) [inline backend/requirements.txt]: Please remove duplicate packages

Comment (42Max42) [reply backend/requirements.txt]: fixed

**Comment by 42Max42** [comment]
@jsbroks I changed to yarn because I wasn't able to build
`Step 4/19 : RUN npm install -g --quiet     @vue/cli@3.3.0     @vue/cli-service@3.3.0
 ---> Running in c028bd0fbca2
npm ERR! Unexpected end of JSON input while parsing near '....3","chalk":"^2.4.1",'`
I created the db folder for convenience in case of transfer between my machines.

**Comment by jsbroks** [comment]
I think its best to figure out why npm is failing before we merge this in

**Comment by 42Max42** [comment]
Hi, npm was failing because of the repositories, (but not yarn). If you want to only use npm it is a bit tricky (but possible).

**Comment by jsbroks** [comment]
What repository was failing? I dont mind switch if we have a reason too. I use yarn in all my other projects anyway.

**Comment by 42Max42** [comment]
I reverted to the original npm, i changed of network (the original one has an uncertain behavior).

**Comment by 42Max42** [comment]
@jsbroks the vue packages were failing on my network.

**Pull request merged**